From c87faf380ddf44da9d624dabd28178c9065f0f76 Mon Sep 17 00:00:00 2001
From: Adam Duskett <aduskett@gmail.com>
Date: Mon, 3 Feb 2020 10:07:15 -0800
Subject: [PATCH] disable tests

If introspection data on the host is not built, meson throws the error:
"Unknown variable "typelibs". Because tests are not required, removing
the subdir tests alltogether fixes this issue.

Signed-off-by: Adam Duskett <aduskett@gmail.com>
---
 meson.build | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/meson.build b/meson.build
index c2cb577f..65ce7adf 100644
--- a/meson.build
+++ b/meson.build
@@ -237,12 +237,6 @@ else
 endif
 subdir('docs')
 
-# The tests will also run, which is not possible if they
-# were built for a different architecture.
-if not meson.is_cross_build()
-  subdir('tests')
-endif
-
 install_data('Makefile.introspection', install_dir: join_paths(get_option('datadir'), 'gobject-introspection-1.0'))
 install_data('m4/introspection.m4', install_dir: join_paths(get_option('datadir'), 'aclocal'))
 
-- 
2.25.1

